src/styles/blocks/calendar.scss

```
@@ -0,0 +1,57 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.calendar {
+  display: flex;
+  align-items: center;
```

```suggestion
  align-items: center;
  justify-content: center;
```
calendar should be in center also horizontally